contrib_bots/lib/DefineBot/DefineBot.py

```diff
@@ -0,0 +1,72 @@
+# See readme.md for instructions on running this code.
+import json
+import requests
+
+class DefineHandler(object):
+    '''
+    This plugin define a word that the inputs. It
```

Comment:
I think something is missing here - I'd say: `This plugin defines a word that the user inputs`.

On [line 39](https://github.com/zulip/zulip/pull/2757/files#diff-b99a7a8bf879fbff48ba61acc1943e16R39) you mention a word or phrase - is there a limit for the length of the search query? If so, I don't think you're checking it at the moment.

I've tried to define a few phrases, but it didn't work - could you provide an example of the search query that's more than one word?